content/es/docs/topics/charts.md

```diff
@@ -0,0 +1,1155 @@
+---
+title: "Charts"
+description: "Explica el formato del chart y proporciona una guía básica para crear charts con Helm."
+aliases: [
+  "docs/developing_charts/",
+  "developing_charts"
+]
```

Comment:
Aliases are only needed if a page is renamed. You can remove lines 4 through 7 here. (This would incorrectly move English-page readers to looking at this page in Spanish.)